scripts/issue/651/rate_emissions_b80BADGER-20WBTC.py

```diff
@@ -0,0 +1,12 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+def main():
+    techops = GreatApeSafe(registry.eth.badger_wallets.techops_multisig)
+
+    emission_control = techops.contract(registry.eth.EmissionControl)
+
+    emission_control.setTokenWeight(registry.eth.sett_vaults["b80BADGER-20WBTC"], 8000)
```

Comment:
indeed, dashes break the access through dotmap, personally would opt for an underscore here for that reason.